functions/src/extract.ts

```diff
@@ -50,7 +50,8 @@ const getPayload = (snapshot: DocumentSnapshot) => {
     logs.debug('field', field);
     logs.debug('value', value);
 
-    if (value) {
+    // for fields with boolean data type, the value of false is a valid value.
+    if (value || value == false) {
```

Comment:
```suggestion
    // for fields with boolean data type, the value of false is a valid value.
    if (typeof value !== 'undefined') {
```

does this also test the same?